test/conftest.py

```diff
@@ -16,6 +17,7 @@
 from snowflake.connector.compat import IS_WINDOWS
 from snowflake.sqlalchemy import URL, dialect
 from sqlalchemy import create_engine
+from sqlalchemy.engine.mock import create_mock_engine
```

Comment:
In test/README.rst, to install 'mock' has to add to the instructions.

Reply:
obsolete as MockEngine was removed